FrontEnd/main/src/components/landing-page/divider/Divider.js

```diff
@@ -0,0 +1,7 @@
+import './Divider.css';
+
+export const MenuDivider = () => {
+  return (
+      <div className="divider"/>
```

Comment:
Do we really need a separate component for this?

Reply:
Divider component removed